app/javascript/react/components/App.js

```diff
@@ -1,15 +1,15 @@
-import React from "react";
-import { ApolloProvider } from "@apollo/client";
-import ApolloClient from "../../api/ApolloClient";
-import ApiDemo from "./ApiDemo";
+import React from 'react';
+import { ApolloProvider } from '@apollo/client';
+import ApolloClient from '../../api/ApolloClient';
+import MainComponent from './MainComponent/MainComponent';
+import ApiDemo from './ApiDemo/ApiDemo';
 
 const App = () => (
   <ApolloProvider client={ApolloClient}>
     <div>
```

Comment:
we probably don't need the wrappingn div

Reply:
removed wrapping div